tests/v0_tests/test_tensor_search.py

```diff
@@ -336,4 +335,56 @@ def test_multi_queries(self):
             print(res)
             # the poodle doc should be lower ranked than the irrelevant doc
             for hit_position, _ in enumerate(res['hits']):
-                assert res['hits'][hit_position]['_id'] == expected_ordering[hit_position]
\ No newline at end of file
+                assert res['hits'][hit_position]['_id'] == expected_ordering[hit_position]
+
+    def test_escaped_non_tensor_field(self):
+        """We need to make sure non tensor field escaping works properly.
+
+        We test to ensure Marqo doesn't match to the non tensor field
+        """
+        docs = [{
+            "dont#tensorise Me": "Dog",
+            "tensorise_me": "quarterly earnings report"
+        }]
+        self.client.index(index_name=self.index_name_1).add_documents(
+            docs, auto_refresh=True, non_tensor_fields=["dont#tensorise Me"]
+        )
+        if self.IS_MULTI_INSTANCE:
+            self.warm_request(self.client.index(self.index_name_1).search, q='Blah')
+        search_res = self.client.index(index_name=self.index_name_1).search("Dog")
+        assert list(search_res['hits'][0]['_highlights'].keys()) == ['tensorise_me']
+
+    def test_special_characters(self):
```

Comment:
isn't `test_escaped_non_tensor_field` just a specific test case for `test_special_characters`. i.e. do we need `test_escaped_non_tensor_field`?